test/unit/YawPitchControl/YawPitchControl.spec.js

```diff
@@ -203,6 +221,23 @@ describe("YawPitchControl", function() {
 				done();
 			}
 		});
+
+		it("should lookAt trigger change event once when enable called multiple times.", () => {
+			// Given
```

Comment:
Is this case for checking axes not connected PanInput multiple times?

Reply:
Yes! Exactly.

Reply:
OK Thank you!